core/frontend/src/App.vue

```diff
@@ -630,12 +630,8 @@ export default Vue.extend({
       }
     },
     createExtensionAddress(service: Service): string {
-      if (service?.metadata?.new_page !== true) {
-        return `/extensions/${service.port}`
-      }
-
-      // The `//` force href to be used as absolute path and not as relative
-      return `//${window.location.hostname}:${service.port}`
+      const sanitized_name = service?.metadata?.name.toLowerCase().replace(/[^a-z0-9]/g, '')
```

Comment:
Since this is being done is helper, why not putting the path on the API ? With that we'll have less hardcoded references and we can deal with this path in others places, and avoiding doing regex around since the API will be the source of true.